### DISCUSSION TITLE
What's currently correct way of changing coordinators - conbee 2 to cc2652rb

### DISCUSSION THREAD

**Discussion by bednim** [Koenkk/zigbee2mqtt#12298] [Devices]
Hi, I'm changing coordinators from conbee 2 to cc2652rb - what's currently best way of doing so?

Thanks!

**Comment by bednim** [comment]
Anyone please? Thanks!

**Comment by lux73** [comment]
if you search the forums it seems Coordinator change/replace is not trivial and could lead in weird behaviour:

https://github.com/Koenkk/zigbee2mqtt/discussions?discussions_q=coordinator+change
https://github.com/Koenkk/zigbee2mqtt/discussions?discussions_q=coordinator+replace

i would recommend to re-build your Zigbee Network from scratch
